.github/workflows/rust.yml

```diff
@@ -11,6 +11,22 @@ on:
     paths-ignore:
       - 'README.md'
 jobs:
+  fmt:
+    name: Rustfmt
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions-rs/toolchain@master
+        with:
+          profile: minimal
+          toolchain: stable
+          components: rustfmt
+          override: true
+      - uses: actions-rs/cargo@master
+        with:
+          command: fmt
+          toolchain: stable
+          args: --all -- --check --config merge_imports=true
```

Comment:
Can you create a `.rustfmt.toml` file and put the config there?

Reply:
done